src/tiles/list/tile-list-compact.ts

```diff
@@ -43,6 +49,14 @@ export class TileListCompact extends LitElement {
     `;
   }
 
+  // Only in list, not tile
+  private get collectionColor() {
```

Comment:
q: i see that `collectionColor` is defined here and also in `src/tiles/list/tile-list.ts`, can it be just 1?

Reply:
A: great question! In order for it to be a `get`, I think it would have to be in a 'list' superclass and inherited. I could make it a directive, or an outside function like `src/tiles/list/account-label.ts`

I agree it should be DRY'ed up. I'll take a look at a directive again, to see it it's appropriate

Reply:
Is there a reason we're using a different color for the collection icon on these tiles?

Reply:
There are a number of repeated elements between 'compact' and 'detailed' lists where it seems a mixin would work to remove the duplications -- i.e. a 'list' superclass, in essence, as noted above

Reply:
https://lit.dev/docs/composition/mixins/#mixins-in-typescript